ansible_collections/arista/cvp/docs/modules/cv_validate_v3.rst.md

```diff
@@ -0,0 +1,98 @@
+# cv_validate_v3
+
+Online/Offline configlet validation
```

Comment:
Maybe we should add an explanation of what online and offline validation is here?

Reply:
I can change it to `CVP/Local configlet Validation`